src/Custom.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
 	<PropertyGroup>
-		<MinVerMinimumMajorMinor>0.1</MinVerMinimumMajorMinor>
+		<MinVerMinimumMajorMinor>2.2</MinVerMinimumMajorMinor>
 		<MinVerAutoIncrement>minor</MinVerAutoIncrement>
```

Comment:
This is not what's documented, I think I need to get the 👌 from @bording to make this change.

Reply:
In this case, I think release-2.1 branch is incorrect, it should be `patch` as well ( [based on the docs](https://github.com/Particular/Platform/blob/main/guidelines/branching-strategy.md#releasing-a-new-major-or-minor-version)).  Had it been correct, we wouldn't have had to change it because it would have already been set to `patch`